.github/workflows/nightly-release.yml

```diff
@@ -204,9 +204,11 @@ jobs:
         env:
           RUST_BACKTRACE: 1
         run: |
+          sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 25088A0359807596
+          echo "deb http://ppa.launchpad.net/pipewire-debian/pipewire-upstream/ubuntu $(lsb_release -cs) main" | sudo tee -a /etc/apt/sources.list.d/pipewire-upstream.list
           sudo add-apt-repository universe
```

Comment:
this technically replaces command above but i kept it same as on release, later we might need to look at nightly/stable code and make them more or less the same